torchacc/dist/fsdp.py

```diff
@@ -202,3 +204,151 @@ def clip_grad_norm_(self, max_grad_norm):
 
     def forward(self, *args, **kwargs):
         return self.model(*args, **kwargs)
+
+    def optim_state_dict(self,
+                         optim: torch.optim.Optimizer,
+                         full_state_dict: str = 'FULL_STATE_DICT',
```

Comment:
This is best as bool type, or the name is changed to state_dict_type. Do not use strings that are easy to misspell, it is better to use enum types.

Reply:
I use a enum type StateDictType now